flutter/test/sentry_flutter_test.dart

```diff
@@ -71,7 +71,7 @@ void main() {
       final event =
           verify(transport.send(captureAny)).captured.first as SentryEvent;
 
-      expect(event.sdk.integrations.length, 5);
+      expect(event.sdk.integrations.length, 6);
```

Comment:
I should have fixed those using `whereType` instead, checking the `length` is suboptimal :(